news/2020-09-05-mappers-report-august.md

```diff
@@ -0,0 +1,186 @@
+- **For osu!catch by [Baron](https://osu.ppy.sh/users/10286499)**:
+The osu!catch team is looking for some new and existing modders to break out of their shell and dive deeper into the intricacies of modding. With well known experienced modders and BN's you can be guaranteed the best learning experience and take your modding to the next level.
+- **For osu!mania by [Davvy](https://osu.ppy.sh/users/10047413)**: 
+Mania has recently been seeing a steady increase in both new modders, mappers and Beatmap Nominators alike, and it is time to take that to the next level. With the start of this project we're looking forward towards helping mania mappers become proficient modders in any keymode of their wishes. Our mentor team will be able to handle every aspect of mania modding such as SV, layering, patterning, spread, timing and whatever else is relevant. 
+With that being said, we hope to create a new series of great modders helping out the overall quality of maps out there, mania definitely needs it especially for unpopular keymodes. 
+
+Have we got your interest? Or do you have more questions regarding the project? Then please do not hesitate to join us at the [Modding Mentorship discord server](https://discord.gg/TyvrMvR)
+
+### The osu! Map Cast
+
+Mentioned previously in the Mappers' Report, the **osu! Map Cast** has recently concluded its first season with a whopping **23 episodes in total**.  This podcast hosted by [Bailie](https://osu.ppy.sh/users/11868659) (previously known as Cinnabun) features conversations with a wide variety of figures in the mapping and modding community, such as mappers both old and new, BNs, NAT, and even recently Ephemeral! The podcast series cover a huge variety of topics, both within and outside of osu! mapping- there's even an episode about Beat Saber in there! If you want to learn more about the osu! community, its members, or just want a chill podcast to listen to in the background, be sure to check out the full episode listing on their [forum post](https://osu.ppy.sh/community/forums/topics/1043221)!
+
+<a id="conclusion"></a>
+
+---
+
+And with that, we conclude the Mappers' Report for this month. Again a bit of a read but we're happy you made it until here. 
+
+As always, if you have anything noteworthy we could include in the next issue of the Mappers' Report, please do not hesitate to tell [Feerum](https://osu.ppy.sh/users/4815717) or [Noffy](https://osu.ppy.sh/users/1541323). You could as well join us in the #mappers-report channel on the [osu! Community Discord Server](https://discord.gg/0Vxo9AsejDkGlk3H).
+
+Hope to see you there soon, thank you for reading and stay safe and healthy!
```

Comment:
not sure about this one, but still
```suggestion
Hope to see you there soon, thank you for reading, and stay safe and healthy!
```

Reply:
I think i leave this one out. I use the same sentence since some months and it never really was an issue